website_form_metadata/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+=====================
+Website Form Metadata
+=====================
+
+Odoo has the hidden and unused ability to gather metadata from the website
+forms into the model chatter that that form writes into. This module unveils
```

Comment:
s/that that/that

Reply:
Well, you know that "that that" that bugs you isn't exactly incorrect although a little bit cacophonous :smile: I've rephrased it.